### ISSUE TITLE
Keyboard navigation for "More/Less" button on ColorPicker is out of order

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/microsoft-ui-xaml#8883]
### Describe the bug

When tab-navigating through the ColorPicker when `IsMoreButtonVisible` is enabled, the More/Less button appears in a non-standard order. The user is expecting to move left-right, top-down, but navigating between the RGB combobox and the More/Less button is weirdly out of order.

### Steps to reproduce the bug

1. Go to winui2gallery://item/ColorPicker
2. Enable `IsMoreButtonVisible` option
3. Click on ColorPicker
4. Tab navigate through it in it's expanded form (and tab navigate backwards, if you want)

### Expected behavior

Tab order should be...
1. 2D pane
2. slider
3. "Less" btn
4. RGB combobox
5. etc...

BUG: 3 and 4 are switched

### NuGet package version

WinUI 2 - Microsoft.UI.Xaml 2.8.2

### Windows version

Windows Insider Build (xxxxx)
